src/redhatTelemetry.ts

```diff
@@ -0,0 +1,19 @@
+import { getTelemetryService, TelemetryEvent, TelemetryService } from "@redhat-developer/vscode-redhat-telemetry";
+
+export async function record(eventName: string, properties?: object) {
+  const telemetryService: TelemetryService = await getTelemetryService("redhat.fabric8-analytics")
+  let event:TelemetryEvent={
+    type: 'track',
+    name: eventName,
+    properties: properties
+  }
+  telemetryService.send(event);
+}
+export async function startUp() {
+  const telemetryService: TelemetryService = await getTelemetryService("redhat.fabric8-analytics")
+  telemetryService.sendStartupEvent();
```

Comment:
Isn't `telemetryService.sendStartupEvent();` returning a promise which need to be either awaited or returned?

Reply:
done